src/features/DisplayControls/DrugChart/components/DrugChartView.jsx

```diff
@@ -131,7 +133,9 @@ export default function DrugChartWrapper(props) {
     updatedStartEndDates({ startDate: startDateTime, endDate: endDateTime });
     callFetchMedications(startDateTime, endDateTime);
   };
-
+  const dateFormatString = drugChart.enable24HourTime
+    ? "DD/MM/YYYY HH:mm"
+    : "DD/MM/YYYY hh:mm A";
```

Comment:
can we move it to drugChartUtils as this way it need not to be rendered every time

Reply:
done